tests/components/lock/test_demo.py

```diff
@@ -48,3 +49,10 @@ def test_unlocking(self):
         self.hass.block_till_done()
 
         self.assertFalse(lock.is_locked(self.hass, FRONT))
+
+    def test_opening(self):
+        """Test the opening of a lock."""
+        lock.open(self.hass, OPENABLE_LOCK)
+        self.hass.block_till_done()
+
+        self.assertFalse(lock.is_locked(self.hass, OPENABLE_LOCK))
```

Comment:
Mock also the service call and look if they will be called

Reply:
Do you have an example for me how to do this?

Reply:
https://github.com/home-assistant/home-assistant/blob/dev/tests/common.py#L193